src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+#block-container {
```

Comment:
don't style with ids, they have quite a high specificity so they are harder to override with other styles